MIAUI/MIAUI/ViewModels/ProfilePageViewModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace MIAUI.ViewModels;
+public partial class ProfilePageViewModel
+{
```

Comment:
Since this ViewModel is empty and not being used anywhere, we can delete this file!